Tensile/KernelWriterAssembly.py

```diff
@@ -8459,7 +8472,7 @@ def cleanupGlobalWrite(self, kernel):
   ##############################################################################
   # Return max global write vector width, in elements
   def maxGwvw(self, kernel):
-    atomic = kernel["GlobalSplitU"] > 1
+    atomic = (kernel["GlobalSplitU"] > 1) and (kernel["_GlobalAccumulation"] != 2)
```

Comment:
would it be better to specifically choose _GlobalAccumulation ==1 in case GSUalgo is growing? what can be considered is put atomic condition to a reusable function as some other places use this as well

Reply:
0 and 1 need to use atomic